components/brave_wallet_ui/panel/container.tsx

```diff
@@ -412,10 +413,6 @@ function Container (props: Props) {
           <ConfirmTransactionPanel
             onConfirm={onConfirmTransaction}
             onReject={onRejectTransaction}
-<<<<<<< HEAD
```

Comment:
Please interactive rebase this it out in the original commit.

Reply:
done